app/Http/Controllers/UserController.php

```diff
@@ -28,11 +28,18 @@ public function store(Request $request)
             'description' => '',
         ]);
 
-        User::create($request->all());
+        if(User::where(['email' => $request['email']])->first() != null)
```

Comment:
- The check if the email is already in the database can be done using the `validate` function just above.
  - See : 
    - https://laravel.com/docs/8.x/validation#rule-unique
    - And this for an example : https://laravel.com/docs/8.x/validation#creating-form-requests